.github/workflows/nodejs.yml

```diff
@@ -24,12 +24,6 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
 
-      - name: 'Cache dependencies'
-        uses: 'actions/cache@v2.1.4'
-        with:
-          path: '**/node_modules'
-          key: ${{ runner.os }}-${{ hashFiles('**/package.json') }}
-
       - run: 'npm install'
```

Comment:
I would use `npm ci` instead of `npm install` for ci workflow, https://docs.npmjs.com/cli/v7/commands/npm-ci